pinot-common/src/main/java/com/linkedin/pinot/common/config/TableNameBuilder.java

```diff
@@ -18,52 +18,106 @@
 import com.google.common.base.Preconditions;
 import com.linkedin.pinot.common.utils.CommonConstants.Helix.TableType;
 import com.linkedin.pinot.common.utils.SegmentName;
-import com.linkedin.pinot.common.utils.StringUtil;
 import javax.annotation.Nonnull;
+import javax.annotation.Nullable;
 
 
 public class TableNameBuilder {
-
   public static final TableNameBuilder OFFLINE_TABLE_NAME_BUILDER = new TableNameBuilder(TableType.OFFLINE);
```

Comment:
Rename this to OFFLINE/REALTIME, so that it looks like `TableNameBuilder.OFFLINE` instead of the longer name.